apiary.apib

```diff
@@ -3315,6 +3317,151 @@ Get information about a meeting.
+        + customer (object, required)
+            + type (enum, required)
+                + Members
+                    + company
+                    + contact
+            + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+        + due_date: `2016-02-04T16:00:00+00:00` (string, required)
+        + responsible_user_id: `98b2863e-7b01-4232-82f5-ede1f0b9db22` (string, required)
+        + custom_fields (array[CustomFieldValue], optional)
+
++ Response 201 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + type: `call` (string)
+            + id: `65a35860-dcca-4850-9fd6-47ff08469e0c` (string)
+
+### calls.list [POST /calls.list]
+
+Get a list of calls.
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + start_date: `2021-11-21` (string) - Filter on dates that are on or after the given day
+            + end_date: `2022-05-13` (string)  - Filter on dates that are on or before the given day
+            + relates_to (object, optional) - Filter calls by related object (currently only companies allowed)
+                + id: `415fa7e0-e319-0952-b973-f067f2b49d56` (string)
+                + type: `company` (string)
+            + milestone_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Filter calls by milestone
+            + call_outcome_id: `6bd20dd8-dc3f-0e68-8919-dee0be6efe55` (string, optional) - Filter on done calls by outcome
+        + page (object, optional)
+            + size: `20` (number, optional) - The amount of entries returned per request, max 1 to 100
+                + Default: `20`
+            + number: `1` (number, optional) - The current page, the first page is 1
+                + Default: `1`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + added_at: `2021-11-21T18:14:15+00:00` (string, nullable)
+                + customer (object, nullable)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                    + type: `company` (string)
+                + description: `Call with customer` (string, nullable)
+                + optional_contact (object, nullable)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                    + type: `contact` (string)
+                + outcome (object, nullable)
+                    + type: `callOutcome` (string)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + outcome_summary: `Called, but was not available` (string, nullable)
+                + responsible_user (object, nullable)
+                    + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+                    + type: `user` (string)
+                + scheduled_at: `2021-11-21T19:14:15+00:00` (string)
```

Comment:
Q: Is this the time for when the call should take place? If that is the case I think it must be `on` or `for` and never `at`.

Reply:
> Q: Is this the time for when the call should take place? If that is the case I think it must be on or for and never at.

We use `on` for dates and `at` for date+time, so I don't see any issue in the naming here.

Reply:
Topic was discused on internal channels.

I'm okay if we stay on `scheduled_at`